client/components/data/query-plugin-lists/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { fetchPluginsList } from 'state/plugins/wporg/actions';
+
+class QueryPluginLists extends Component {
+	componentWillMount() {
+		this.request( this.props );
+	}
+
+	request( props ) {
+		props.categories.forEach( ( category ) => {
+			this.props.fetchPluginsList( category );
```

Comment:
warning here - we're using `this.props` on this line but using the passed-in props in the rest of the method